src/main/app/Resources/modules/data/types/fields/modals/parameters/components/parameters.jsx

```diff
@@ -113,6 +169,43 @@ class ParametersModal extends Component {
                 displayed: !this.state.restrictions.locked || this.state.restrictions.lockedEditionOnly
               }
             ]
+          }, {
+            id: 'conditions',
+            icon: 'fa fa-fw fa-cog',
+            title: trans('conditional_rendering'),
+            fields: [
+              {
+                name: 'conditions.dependencyField',
+                type: 'choice',
+                label: trans('dependency_field'),
+                onChange: (value) => {
+                  this.updateConditions('comparisonValue', null)
+                  this.updateConditions('dependencyField', value)
+                },
+                options: {
+                  choices: this.mapFieldsToOptions(this.getDependencyFields(this.getAllFields())),
+                  condensed: true
+                }
+              }, {
+                name: 'conditions.validationType',
+                type: 'choice',
+                label: trans('validation_type'),
+                onChange: (value) => this.updateConditions('validationType', value),
+                options: {
+                  choices: {[EQUALS_TYPE]: trans('equals'), [DOES_NOT_EQUAL_TYPE]: trans('does-not-equal')},
+                  condensed: true
+                }
+              }, {
+                name: 'conditions.comparisonValue',
```

Comment:
Must be : 
- in the `linked` section of `conditions.dependencyField`
- only displayed if there is a field selected
- required